src/Template/Bake/Template/add.ctp

```diff
@@ -0,0 +1,76 @@
+<%
```

Comment:
Bake templates using erb style `<%` tags are deprecated. The new templates need to use twig syntax.

Reply:
That will make migration a bit more difficult, also if stuff needs to be backported, fixes etc.
I wonder if we should continue using the deprecation in this case.
A rewrite in twig makes more sense for complete new projects or plugins and their bake templates IMO at this point.
The old style wont go anywhere soon - especially during 3.x.

Reply:
I don't see any migration issue in relation to bake templates. When baking one doesn't really care what syntax the templates use. If someone has overridden the templates they are going to have to update them anyway for bootstrap 4 and in that case they can use which syntax they prefer.